src/autoupdater.ts

```diff
@@ -122,7 +122,15 @@ export class AutoUpdater {
       mergeOpts.commit_message = mergeMsg;
     }
 
-    await this.merge(mergeOpts);
+    try {
+      await this.merge(mergeOpts);
+    } catch (e) {
+      ghCore.error(
+        `Caught error running merge, skipping and continuing with remaining PRs`,
+      );
+      ghCore.setFailed(e);
```

Comment:
is this necessary if we're already logging out the error in the merge method?

Reply:
Yea - `setFailed` sets the process exitcode which marks this run as failed (which in turn sends an email to the repository / PR owner)

Reply:
cool lgtm